tensorflow/lite/micro/micro_profiler.h

```diff
@@ -72,6 +76,14 @@ class MicroProfiler {
   uint32_t end_ticks_[kMaxEvents];
   int num_events_ = 0;
 
+  struct ticks_per_tag {
```

Comment:
rename: TicksPerTag

https://google.github.io/styleguide/cppguide.html#Type_Names

Reply:
Thank you for the link! Really helpful.